src/swupd_internal.h

```diff
@@ -17,5 +17,36 @@ extern enum swupd_code install_main(int argc, char **argv);
 extern enum swupd_code repair_main(int argc, char **argv);
 extern enum swupd_code diagnose_main(int argc, char **argv);
 extern enum swupd_code bundle_info_main(int argc, char **argv);
+extern enum swupd_code third_party_main(int argc, char **argv);
+extern int third_party_add(void);
+extern int third_party_list(void);
+extern int third_party_remove(void);
+
+struct subcmd {
+	char *name;
+	char *doc;
+	enum swupd_code (*mainfunc)(int, char **);
+};
+
+static inline int subcmd_index(char *arg, struct subcmd *commands)
+{
+	struct subcmd *entry = commands;
+	int i = 0;
+	size_t input_len;
+	size_t cmd_len;
+
```

Comment:
why did you remove this from the function?
```
if (strcmp(arg, "verify") == 0) {		
 	verify_set_command_verify(true); // set to true so we know the "verify" command was used		
 	arg = "diagnose";		
 }
```

Reply:
see #1180

Reply:
just did, that explains it, thanks

Reply:
Marking this as resolved.